package.json

```diff
@@ -56,8 +56,9 @@
     "eslint-plugin-react-native": "^3.10.0",
     "jest": "^26.6.3",
     "prettier": "^2.2.1",
-    "react-native": "^0.63.4",
-    "react-test-renderer": "^17.0.1",
+	  "react": "^18.0.0",
+    "react-native": "^0.68.0",
```

Comment:
phew this is a bit of a risky change, don't you think? do we need to update some other template-specific stuff?

Reply:
I'm fine changing it to whatever version. The types in this repo don't check against a whole range of versions (and setting that up would be a serious PITA). I figured latest was fine, since peer dependencies are "*"